dso-l2/src/main/java/com/tc/l2/state/ElectionManagerImpl.java

```diff
@@ -249,16 +252,16 @@ private synchronized void electionStarted(Enrollment e, State serverState) {
     logger.info("Election Started : " + e);
   }
 
-  private NodeID doElection(NodeID myNodeId, boolean isNew, WeightGeneratorFactory weightsFactory, State currentState)
+  private NodeID doElection(NodeID myNodeId, Topology topology, boolean isNew, WeightGeneratorFactory weightsFactory, State currentState)
       throws GroupException, InterruptedException {
 
     // Step 1: publish to cluster NodeID, weight and election start
     Enrollment e = EnrollmentFactory.createEnrollment(myNodeId, isNew, weightsFactory);
-    electionStarted(e, currentState);
+    electionStarted(e, currentState, topology);
 
     L2StateMessage msg = L2StateMessage.createElectionStartedMessage(e, currentState);
     debugInfo("Sending my election vote to all members");
-    groupManager.sendAll(msg);
+    groupManager.sendTo(topology.getServers(), msg);
```

Comment:
use the local server list that you will generate at the beginning here.

Reply:
Separate local server list won't be necessary as Topology class is immutable and we don't need to change much when voter count changes supported.